microsoft-edge/devtools-guide-chromium/whats-new/2019/12/devtools.md

```diff
@@ -344,7 +344,9 @@ If you are on Windows or macOS, consider using the [Microsoft Edge preview chann
 
 [AccessibilityInsights]: https://aka.ms/a11yinsights "Accessibility Insights"  
 
-[DwarfHome]: https://dwarfstd.org "Dwarf Home"  
+[DwarfHome]: http://dwarfstd.org "Dwarf Home"  
+<!-- this link must be http, not https -->  
```

Comment:
This is the right place to link to but since it's not https, I say we just cut the link

Reply:
Removed.